src/options/options_handler.cpp

```diff
@@ -1178,14 +1147,12 @@ theory::bv::BitblastMode OptionsHandler::stringToBitblastMode(
     if (!options::bitvectorAlgebraicSolver.wasSetByUser()) {
       options::bitvectorAlgebraicSolver.set(true);
     }
+    if (options::bvSatSolver() != theory::bv::SAT_SOLVER_MINISAT)
+    {
+      throwLazyBBUnsupported(options::bvSatSolver());
+    }
     return theory::bv::BITBLAST_MODE_LAZY;
   } else if(optarg == "eager") {
-
```

Comment:
Is this PR supposed to be merged after #1838?

Reply:
Oh, I totally forgot that incremental eager is not yet merged in :-). Yes this should come after #1838 is merged.